crates/bevy_window/src/event.rs

```diff
@@ -47,3 +47,10 @@ pub struct ReceivedCharacter {
     pub id: WindowId,
     pub char: char,
 }
+
+/// An event that indicates a window has received or lost focus.
+#[derive(Debug, Clone)]
+pub struct FocusedWindow {
```

Comment:
Since it can be focused or unfocused, maybe a different name may be valuable. Something like `WindowFocus` or `FocusChanged`?

Reply:
I personally like `WindowFocus`